### PR TITLE
feat(pipelinetemplate): Add support for list injection for Before and After stanzas

### PR DESCRIPTION
**Pull request by jeyrschabu**
- Before and After injections now support a list of target stage Ids

### CODE REVIEW

**Comment by jeyrschabu** [comment]
@robzienert PTAL whenever you get back :)

File: orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/graph/transform/ConfigStageInjectionTransform.java [hunk 1]
```diff
@@ -117,11 +119,22 @@ private static void dfs(String stageId,
 
     state.put(stageId, Status.VISITING);
     StageDefinition stage = graph.get(stageId);
+    if (stage == null) {
```
Comment (robzienert) [inline orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/graph/transform/ConfigStageInjectionTransform.java]: This happens? Is it worthwhile logging this?

**Comment by robzienert** [comment]
👍 Looks right to me, one minor question - but can be merged without after rebase.

File: orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/graph/transform/ConfigStageInjectionTransform.java (hunk 1 above)
Comment (jeyrschabu) [inline orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/graph/transform/ConfigStageInjectionTransform.java]: You know what ? I think that's me being lazy on rebuilding the graph after it has been mutated. I think I can get rid of that null check and handle rebuilding the graph. Keeps the dfs method clean that way

**Pull request merged**